ui/admin/tests/acceptance/storage-buckets/create-test.js

```diff
@@ -58,6 +57,11 @@ module('Acceptance | storage-buckets | create', function (hooks) {
     await fillIn(commonSelectors.FIELD_NAME, commonSelectors.FIELD_NAME_VALUE);
     await select(selectors.FIELD_SCOPE, 'global');
 
+    await fillIn(
```

Comment:
can we also validate that after save is clicked, the codeblock is shown (not the editor)? Or will the read type tests be added in a different PR?

Reply:
Sorry, I should have called that out in the description. I was going to go back through the tests and add those in afterwards since the focus of this PR is "create".